spec/support/shared_examples/proofer.rb

```diff
@@ -1,11 +1,13 @@
 shared_examples 'a proofer' do
   let(:verification_status) { 'passed' }
+  let(:conversation_id) { 'foo' }
   let(:verification_errors) { {} }
   let(:result) { Proofer::Result.new }
 
   before do
     response = instance_double(LexisNexis::Response)
     allow(response).to receive(:verification_status).and_return(verification_status)
+    allow(response).to receive(:conversation_id).and_return(conversation_id)
```

Comment:
can we add a part of the spec that reads back the transaction ID we set from this?